primitives/linux-port/primitiveprocessor.h

```diff
@@ -492,14 +493,17 @@ T getNullValue(uint8_t type)
     case execplan::CalpontSystemCatalog::BLOB:
     case execplan::CalpontSystemCatalog::TEXT: return joblist::CHAR8NULL;
 
+    // VARCHARs with width >= 8 are stored as dictionaries
```

Comment:
Same is for CHAR type. Please add it here and also add appropriate column into a test case.

Reply:
Thanks, added CHAR